### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Edit-Web Application_Keyboard: "Pipeline" and ".Net framework" list items are not accessible using the keyboard.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#165]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated**: MAS2.1.1

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "More" commands button of the existing website inside the grid and Press Enter.
6. Tab till "Edit" in the expanded menu and Press Enter.
7. Tab till "Web Applications" in the left menu section and Press Enter.
8. Tab till "Create Web Application" button and Press Enter.
9. Navigate to Application Pool page tab.
10.Verify that all the interactive controls of the page are accessible through keyboard.

**Actual Result:**
Keyboard focus is not going to "Pipeline" and ".Net framework" list items and hence they are not accessible using keyboard.

**Expected Result:**
Keyboard focus should move to each interactive control of the page and they should be accessible using the keyboard.

**Note**: Similar issue is observed on "Virtual Directories" page where "Logon Method" list items are not accessible using keyboard.

**User Impact:**
Keyboard only users will be unable to access those controls if they are not accessible using keyboard,. This may affect people who are blind (who cannot use devices such as mice that require eye-hand coordination), people with low vision (who may have trouble finding or tracking a pointer indicator on screen), some people with hand tremors find using a mouse very difficult and therefore usually use a keyboard

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment:**
[Bug 165_List items are not accessible using keyboard.pptx](https://github.com/Microsoft/IIS.WebManager/files/2332106/Bug.165_List.items.are.not.accessible.using.keyboard.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
